src/content/reference/react-dom/preload.md

```diff
@@ -0,0 +1,160 @@
+---
+title: preload
+---
+
+<Intro>
+
+`preload` lets you eagerly fetch a resource such as a stylesheet, font, or external script that you expect to use.
+
+```js
+preload("https://example.com/font.woff2", {as: "font"});
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `preload(href, options)` {/*preload*/}
+
+To preload a resource, call the `preload` function from `react-dom`.
+
+```js
+import { preload } from 'react-dom';
+
+function AppRoot() {
+  preload("https://example.com/font.woff2", {as: "font"});
+  // ...
+}
+
+```
+
+[See more examples below.](#usage)
+
+The `preload` function provides the browser with a hint that it should start downloading the given resource, which can save time.
+
+#### Parameters {/*parameters*/}
+
+* `href`: a string. The URL of the resource you want to download.
+* `options`: an object. It contains the following properties:
+  *  `as`: a required string. The type of resource. Its possible values are `audio`, `document`, `embed`, `fetch`, `font`, `image`, `object`, `script`, `style`, `track`, `video`, `worker`.
```

Comment:
The intended support semantics here are anything that a browser can preload. also the `as` property is named such because this maps to the `as` attribute of a `<link rel="preload"...` tag. Maybe we should clarify where the list of supported types comes from so if/when it evolves it's clear React's implementation will as well?

Maybe link to the relevant MDN documentation?